python/packages/autogen-core/docs/src/user-guide/agentchat-user-guide/index.md

```diff
@@ -111,11 +111,13 @@ tutorial/state
 :hidden:
 :caption: Advanced
 
+tutorial/custom-agents
```

Comment:
we can move the location of `custom-agents.ipynb` to the parent folder. Then add a new redirect entry in the conf.py: https://github.com/microsoft/autogen/blob/main/python/packages/autogen-core/docs/src/conf.py#L175-L176